lightwood/encoders/numeric/ts_numeric.py

```diff
@@ -48,23 +48,18 @@ def encode(self, data, extra_data=None):
                     mean = self._abs_mean
                 if real is not None and mean > 0:
                     vector[0] = 1 if real < 0 and not self.positive_domain else 0
-                    vector[1] = math.log(abs(real)) if abs(real) > 0 else -20
-                    vector[2] = real / mean
+                    vector[1] = real / mean
                 else:
                     log.debug(f'Can\'t encode target value: {real}')
 
             else:
-                vector = [0] * 4
+                vector = [0] * 3
                 try:
-                    if real is None:
-                        vector[0] = 0
-                    else:
+                    if real is not None:
```

Comment:
does real never have an opportunity to be None in the new framework?

Reply:
Good point. It can be `None`. In that case the encoded vector stays zero-filled and when decoding we see its first element is `< 0.5` and we decode `None` back.